notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def creat_bool_tensor(*size):
</pre><p>looks like small typo in function name: should be create_bool_tensor?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done